src/utils/constants.ts

```diff
@@ -3,3 +3,11 @@ export const PAGE_CONFIG = [
   { path: '/prescriptions', title: 'Prescriptions' },
   { path: '/account', title: 'Account' }
 ]
+
+export const STATUS_COLORS = {
+  DANGER: 'danger',
+  WARNING: 'warning',
+  SUCCESS: 'success'
+}
```

Comment:
`COLOR_STATUS`

Reply:
I've changed the object name with COLOR_STATUS